app/src/main/java/com/zulip/android/activities/MessageListFragment.java

```diff
@@ -1,9 +1,5 @@
 package com.zulip.android.activities;
 
-import java.sql.SQLException;
```

Comment:
It's a good idea (when possible) to keep deletion code in the same commit as the code that replaces the functionality (it makes it easier to check that the functionality has been preserved).